scripts/us_bls/jolts/bls_jolts.py

```diff
@@ -0,0 +1,285 @@
+
+  # Drop rate data
+  jolts_df = jolts_df.query("ratelevel_code == 'L'")
+
+  # Drop prelimary data
+  jolts_df = jolts_df.query("footnote_codes != 'P'")
+
+  # Drop non-national data
+  jolts_df = jolts_df.query("region_code == '00'")
+
+  # Map industries
+  def jolts_code_map(row):
+    """ Maps industry code used by BLS Jolts to NAICS or BLS aggregation """
+    industry_code = row['industry_code']
+    assert industry_code in CODE_MAPPINGS
+
+    mapped_code = CODE_MAPPINGS[industry_code]
+
+    # New Jolts code
+    if ":" in mapped_code:
+      JOLTS_CODE = mapped_code.split(":")[0]
+      row['industry_code'] = "JOLTS_" + JOLTS_CODE
+
+    # NAICS
+    else:
+      row['industry_code'] = mapped_code
+    
+    return row
+
+  jolts_df = jolts_df.apply(jolts_code_map, axis=1)
+
+  def map_row_to_statistical_variable(row):
+    """ Maps a row of the dataframe to the Statistical Variable that describes it """
+    base_stat_var = row['statistical_variable']
+    industry_code = row['industry_code']
+    seasonal_adjustment = row['seasonal_adjustment']
+
+    return f"dcs:{base_stat_var}_NAICS_{industry_code}_{seasonal_adjustment}"
+
+  # Build map to Statistical Variable
+  jolts_df['seasonal_adjustment'] = jolts_df['seasonal'].apply(lambda adjustment: "Adjusted" if adjustment == "S" else "Unadjusted")
+  jolts_df['StatisticalVariable'] = jolts_df.apply(map_row_to_statistical_variable, axis=1)
+  jolts_df['Value'] = jolts_df['value']
+
+  return jolts_df, schema_mapping
+
+
+def create_statistical_variables(jolts_df, schema_mapping):
+  """ Creates Statistical Variable nodes.
+
+    A new statistical industry is needed for each of the 6 job variables and for every industry
+    The industry codes may be either NAICS or BLS_JOLTS aggregations. The schema_mapping is used 
+    for additional information for each of the 6 job variables. These new variables are written
+    to the statistical variables mcf file.
+
+    Args:
+      jolts_df: The dataframe of BLS Jolts data created by generate_cleaned_dataframe
+      schema_mapping: The schema mapping created by generate_cleaned_dataframe
+  """
+
+  template_stat_var = """
+  Node: dcid:{STAT_CLASS}_NAICS_{INDUSTRY}_{ADJUSTMENT}
+  typeOf: dcs:StatisticalVariable
+  populationType: {POPULATION}
+  jobChangeEvent: dcs:{JOB_CHANGE_EVENT}
+  statType: dcs:measuredValue
+  measuredProperty: dcs:count
+  measurementQualifier: {BLS_ADJUSTMENT}
+  naics: dcid:NAICS/{INDUSTRY}
+  """
+
+  # Map industry and seasonal adjustment to statistical variable name
+  adjustment_types = [("Adjusted", "dcs:BLSSeasonallyAdjusted"), ("Unadjusted", "dcs:BLSSeasonallyUnadjusted")]
+
+  # Output the schema mapping to a new file
+  with open(STATISTICAL_VARIABLE_FILE, "w+", newline="") as f_out:
+    for schema_name, pop_type, job_change_event, _ in schema_mapping:
+
+        unique_industries = list(jolts_df['industry_code'].unique())
```

Comment:
We can just inline this value in the for loop.

Reply:
Updated.